packages/react-divider/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@fluentui/react-divider",
+  "version": "0.1.0",
+  "description": "React components for building web experiences",
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluentui"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "bundle": "just-scripts bundle",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "just": "just-scripts",
+    "lint": "just-scripts lint",
+    "start": "just-scripts dev:storybook",
+    "start-test": "just-scripts jest-watch",
+    "test": "just-scripts test",
+    "update-snapshots": "just-scripts jest -u"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "^1.0.0-beta.2",
+    "@fluentui/react-conformance": "^1.0.0",
+    "@fluentui/scripts": "^1.0.0",
+    "@testing-library/react": "^10.4.9",
+    "@testing-library/react-hooks": "^5.0.3",
+    "@types/enzyme": "3.10.3",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/react": "16.9.42",
+    "@types/react-dom": "16.9.10",
+    "@types/react-test-renderer": "^16.0.0",
+    "enzyme": "~3.10.0",
+    "enzyme-adapter-react-16": "^1.15.0",
+    "react": "16.8.6",
+    "react-app-polyfill": "~1.0.1",
+    "react-dom": "16.8.6",
+    "react-test-renderer": "^16.3.0"
+  },
+  "dependencies": {
+    "@fluentui/react-hooks": "^8.0.0-beta.14",
+    "@fluentui/react-make-styles": "^0.2.7",
+    "@fluentui/react-theme-provider": "^1.0.0-beta.26",
+    "@fluentui/react-utilities": "^0.3.1",
```

Comment:
Please run syncpack and update your versions

Reply:
`yarn syncpack:fix` 🐱

Reply:
Can this be added to a dev doc somewhere? I wasn't aware of having to do this. However, It has been updated, thanks!

Reply:
This is a problem inherent to creating a new package which references dependencies which are being rapidly bumped (a lot of other converged packages will hit it during development too). @ling1726 didn't you write a doc about convergence package dev process? If so we could add this there.

Reply:
Ran the syncfix and updated